lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/examples/hello/HelloVortexFunction.java

```diff
@@ -30,7 +28,7 @@
    * Prints to stdout.
    */
   @Override
-  public Serializable call(final Serializable serializable) throws Exception {
+  public Object call(final Object object) throws Exception {
```

Comment:
Can we have an example that uses the input and output? 
I get a sense on how unpretty using `Object` is.